tests/installation/bootloader_s390.pm

```diff
@@ -175,10 +180,25 @@ EO_frickin_boot_parms
     $s3270->sequence_3270(qw( String(FILE) ENTER ));
 
     # linuxrc
```

Comment:
linuxrc comment should be in the else, there is not linuxrc in agama.

Reply:
updated.